extensions/amp-form/0.1/amp-form.js

```diff
@@ -351,7 +350,7 @@ export class AmpForm {
       event.preventDefault();
     }
     // Submits caused by user input have high trust.
-    this.submit_(ActionTrust.MEDIUM); // TODO(choumx, #9699): HIGH.
+    this.submit_(ActionTrust.HIGH);
```

Comment:
this should unblock https://github.com/ampproject/amphtml/pull/9879 right? @cvializ